nexus/types/src/external_api/views.rs

```diff
@@ -412,3 +413,47 @@ pub struct DeviceAccessTokenGrant {
 pub enum DeviceAccessTokenType {
     Bearer,
 }
+
+// SYSTEM UPDATES
+
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct SystemVersionRange {
+    // TODO: if we want to use semver::Version, I think we have to newtype it
+    // in order to implement JsonSchema. ew
+    // pub low: semver::Version,
+    // pub high: semver::Version,
+    pub low: String,
+    pub high: String,
+}
```

Comment:
@smklein if we want to use `semver::Version` here, which sounds sensible, we need to wrap it and `impl JsonSchema` on that. Correct?

Reply:
I believe so - looks like someone already started work on this here: https://github.com/GREsau/schemars/pull/195

Reply:
Good timing. I looked but apparently in the wrong place.